sources/datasources/users.js

```diff
@@ -50,11 +49,11 @@ class UserAPI extends DataSource {
 			const createdAccessLevel = await AccessLevel.create(accessLevelObj);
 			await incomingUser.clubAccess.push(createdAccessLevel);
 			await incomingUser.save();
-			await updateJWT(uid,{mongoID:incomingUser._id});
+			if(process.env.NODE_ENV!="test")
```

Comment:
Why does production code need an if-else to check it it is tested?
You can mock this in test code I suppose?

Reply:
Cant quite figure out how to inject updateJWT as a dependency
there are quite a things that could be solved if we could inject dependency from query like the headers, we're still figuring it out, until then this is the only way to go. 
Imma go ahead and close this for now, but i've created this https://github.com/dscnitrourkela/project-dates/issues/74